.github/workflows/reviewdog.yml

```diff
@@ -12,17 +12,6 @@ jobs:
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           reporter: github-pr-review
-  spelling:
```

Comment:
@joe2k01 I'm removing this as it seems to be polluting the PRs with useless comments than actually helping.